### PR TITLE
Add namespace helper

### PR DESCRIPTION
**Pull request by arikarim**
Add namespace and resources router helper

### CODE REVIEW

**Comment by arikarim** [comment]
Fixes #5

File: lib/rage/router/dsl.rb [hunk 1]
```diff
@@ -129,6 +129,54 @@ def match(path, to:, constraints: {}, defaults: nil, via: :all)
       end
     end
 
+    def resources(name, constraints: nil, defaults: nil, only: nil, except: nil)
```
Comment (rsamoilov) [inline lib/rage/router/dsl.rb:132]: Hey @arikarim 

I've worked on the `resources` helper in another PR. Do you want to check it out?

File: lib/rage/router/dsl.rb [hunk 2]
```diff
@@ -129,6 +129,54 @@ def match(path, to:, constraints: {}, defaults: nil, via: :all)
...
+          __on("POST", "#{name}", "#{name}#create", constraints, defaults)
+        when :update
+          __on("PUT", "#{name}/:id", "#{name}#update", constraints, defaults)
+        when :destroy
+          __on("DELETE", "#{name}/:id", "#{name}#destroy", constraints, defaults)
+        end
+      end
+    end
+
+    def namespace(path, &block)
```
Comment (rsamoilov) [inline lib/rage/router/dsl.rb:170]: Can you please add some documentation?

Comment (rsamoilov) [inline lib/rage/router/dsl.rb:170]: It should also support the `path` and `module` options.

https://api.rubyonrails.org/classes/ActionDispatch/Routing/Mapper/Scoping.html#method-i-namespace

File: lib/rage/router/dsl.rb (hunk 1 above)
Comment (arikarim) [reply lib/rage/router/dsl.rb]: Of course,  I will delete the resources part in this one.

**Comment by arikarim** [comment]
Done

**Review by rsamoilov — APPROVED** [review]
Looks good. Thank you.

**Pull request merged**